app/translations/ru.json

```diff
@@ -778,7 +778,7 @@
   "app.containers.ViewQuestion.markGeneralQuestionAndGetEarn":
     "Отметьте лучший ответ в своем вопросе и получите +1 очко рейтинга\"",
   "app.containers.ViewQuestion.generalQuestion": "Общий вопрос",
-  "app.containers.ViewQuestion.expertQuestion": "Экспертный вопрос",
+  "app.containers.ViewQuestion.expertQuestion": "Туториал",
```

Comment:
`expertQuestion` should be changed to `tutorial`

Reply:
changed: https://github.com/peeranha/peeranha-web/pull/652/commits/c6e73e670c1f85b2a6aaaeb31ff347e0a4a110f2